### PR TITLE
Minor tweaks to job descriptions

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About the PR
 A lot of jobs had outdated or misleading information, like doctors talking about viruses and cloning. Now they shouldn't be

## Why / Balance
 Outdated and wrong

## Technical details
None

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

**Comment by Hmeister-real** [comment]
I have literally no idea why the test failed, it is literally just description changes. Maybe because Brigmeds gone?

**Comment by deltanedas** [comment]
test fail erroneous

File: Resources/Locale/en-US/job/department-desc.ftl [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-department-Civilian-description = Perform small helpful tasks to keep the station sane.
+﻿department-Cargo-description = Complete bounties, earn Spessos, and order useful supplies for the crew.
+department-Civilian-description = Perform small helpful tasks to keep the station sane and well catered.
 department-Command-description = Manage the crew and keep them working efficiently.
-department-Engineering-description = Keep the station operational.
+department-Engineering-description = Keep the power on and the station operational.
 department-Medical-description = Keep the crew healthy.
 department-Security-description = Keep the peace around the station.
-department-Science-description = Research new technologies and dangerous artifacts.
+department-Science-description = Research new technologies, dangerous artifacts and anomalies, and Robotics.
```
Comment (TheShuEd) [inline Resources/Locale/en-US/job/department-desc.ftl:7]:
```suggestion
department-Science-description = Research artifacts and anomalies to invent new equipment for the station
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+job-description-doctor = Diagnose and heal crewmembers through medicinal chemicals, advanced medicine, and defibrillators. Make sure the dead don't rot, and that cadavers are in the morgue.
+job-description-engineer = Keep the station's main engine & solars active, optimize the power network, and make emergency repairs using your hardsuit in spaced areas.
 job-description-ertengineer = Ensure that the station has power and clean air.
 job-description-ertjanitor = Ensure that the station is properly cleaned--for morale.
 job-description-ertleader = Lead the Emergency Response Team in dealing with threats to Nanotrasen assets.
 job-description-ertmedic = Ensure that the crew of the station is kept alive and healthy.
 job-description-ertsecurity = Ensure that any active threats to the station are put to rest.
-job-description-hop = Handle access reassignment fairly using your ID console, manage the service department, and keep Ian safe.
+job-description-hop = Handle access reassignment fairly using your ID console, manage demotions, change crews jobs on their request with other heads permissions, manage the service department, and keep Ian safe.
```
Comment (Rainbeon) [inline Resources/Locale/en-US/job/job-description.ftl:27]: These seem like they fall under handling access reassignment fairly already

File: Resources/Locale/en-US/job/job-description.ftl [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
 job-description-hos = Manage your security force and keep them efficient, quell dissent, and keep the other heads of staff safe.
-job-description-intern = Learn the basics of administering medicine, cloning the dead, and creating chemicals.
+job-description-intern = Learn the basics of administering medicine, learning basic chemicals and using basic medical tools.
```
Comment (Rainbeon) [inline Resources/Locale/en-US/job/job-description.ftl:29]: Double mention of learning, the entire sentence structure already revolves around "Learn"

File: Resources/Locale/en-US/job/job-description.ftl [hunk 4]
```diff
@@ -36,10 +36,10 @@ job-description-passenger = Enjoy your stay aboard the station with no obligatio
...
 job-description-rd = Manage the science department, unlocking technologies, acquiring & researching artifacts, and performing experiments.
-job-description-research-assistant = Learn the basics of how to research various artifacts and anomalies.
-job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras and the radio. Currently available on Bagel Station and Core.
-job-description-salvagespec = Use the salvage magnet to draw in derelicts & asteroids to loot and enrich the station, and fight off any space fauna along the way.
-job-description-scientist = Research alien artifacts, unlock new technologies, upgrade machines around the station, and make everything run more efficiently.
+job-description-research-assistant = Learn the basics of how to research various artifacts, anomalies and robotics.
+job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras, the radio and the news. Currently available on Bagel Station and Core.
```
Comment (Rainbeon) [inline Resources/Locale/en-US/job/job-description.ftl:40]: Reporters are also available on Origin

File: Resources/Locale/en-US/job/job-description.ftl [hunk 5]
```diff
@@ -36,10 +36,10 @@ job-description-passenger = Enjoy your stay aboard the station with no obligatio
...
+job-description-salvagespec = Use the salvage magnet to draw in detatched scraps & asteroids to loot and enrich the station, build a salvage ship, and then travel to new planets. and fight off any space fauna along the way.
```
Comment (Rainbeon) [inline Resources/Locale/en-US/job/job-description.ftl:41]: Last sentence has incorrect struct, can't start with a lowercase, perhaps replace with "... build a salvage ship and then travel to new planets, while fighting off any space fauna along the way."

File: Resources/Locale/en-US/job/job-description.ftl [hunk 6]
```diff
@@ -37,8 +37,8 @@ job-description-psychologist = Provide emotional support to traumatized crew. Cu
...
 job-description-rd = Manage the science department, unlocking technologies, acquiring & researching artifacts, and performing experiments.
 job-description-research-assistant = Learn the basics of how to research various artifacts, anomalies and robotics.
-job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras, the radio and the news. Currently available on Bagel Station and Core.
-job-description-salvagespec = Use the salvage magnet to draw in detatched scraps & asteroids to loot and enrich the station, build a salvage ship, and then travel to new planets. and fight off any space fauna along the way.
+job-description-reporter = Entertain & inform the crew with your vibrant journalism through wireless cameras, the radio and the news. Currently available on Bagel Station, Core and Origin.
+job-description-salvagespec = Use the salvage magnet to draw in detatched scraps & asteroids to loot and enrich the station, build a salvage ship, and then travel to new planets, while fight off any space fauna along the way.
```
Comment (Rainbeon) [inline Resources/Locale/en-US/job/job-description.ftl:41]: you missed that "fight" should be "fighting" here

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**